metagpt/configs/rag_config.py

```diff
@@ -0,0 +1,8 @@
+from metagpt.configs.embedding_config import EmbeddingConfig
+from metagpt.configs.query_analysis_config import QueryAnalysisConfig
+from metagpt.utils.yaml_model import YamlModel
+
+
+class RAGConfig(YamlModel):
+    embedding: EmbeddingConfig = EmbeddingConfig()
+    query_analysis: QueryAnalysisConfig = QueryAnalysisConfig()
```

Comment:
It is recommended to add QueryAnalysisConfig and EmbeddingConfig in rag_config.py without hyde_config.py and query_analysis_config.py files.